README.md

```diff
@@ -11,44 +14,78 @@ Computations and statistics on manifolds with geometric structures.
 - The documentation of ```geomstats``` can be found on the [documentation website](https://geomstats.github.io/).
 - If you find ``geomstats`` useful, please kindly cite our [paper](https://arxiv.org/abs/1805.08308).
 
+## Install geomstats via pip3
 
-## Installation
-
-OS X & Linux:
+From a terminal (OS X & Linux), you can install geomstats and its requirements with ``pip3`` as follows::
 
 ```
 pip3 install -r requirements.txt
```

Comment:
First, you don't have the file requirements.txt on your machine to start with. Then I think running `pip3 install geomstats` alone installs all the dependencies and therefore is enough